handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -412,22 +412,30 @@ public String applicationProtocol() {
      *
      * @return the {@link Future} for the iniital TLS handshake if {@link #renegotiate()} was not invoked.
      *         The {@link Future} for the most recent {@linkplain #renegotiate() TLS renegotiation} otherwise.
+     * @deprecated intercept {@link SslHandshakeCompletionEvent}.
      */
+    @Deprecated
     public Future<Channel> handshakeFuture() {
```

Comment:
@trustin hmm true... @Scottmitch makes sense to you as well ? If so let us just close this one.